src/coinbase/webhook.ts

```diff
@@ -163,15 +163,21 @@ export class Webhook {
   }
 
   /**
-   * Updates the webhook with a new notification URI.
+   * Updates the webhook with a new notification URI, and optionally a new list of addresses to monitor.
    *
-   * @param notificationUri - The new URI for webhook notifications.
+   * @param options - The options to update webhook.
+   * @param options.notificationUri - The new URI for webhook notifications.
+   * @param options.eventTypeFilter - The new eventTypeFilter that contains a new list (replacement) of addresses to monitor for the webhook.
```

Comment:
if eventTypeFilter can only contain replacement addresses, should this be called just addresses?

Also `replacement` is confusing. Aren't you adding new addresses to the webhook and not replacing the old ones?

Reply:
1. I thought about the naming `eventTypeFilter`, my thought process is that in future we might add more stuff into `eventTypeFilter`, and we might want to allow clients to update them, so keeping this name for the purpose of extensibility. 
2. It is actually a `PUT` not a `PATCH`, so address array passed in will replace the existing addresses in the webhook. I clarified it in this doc PR https://github.cbhq.net/solutions-architecture/docs/pull/715/files#diff-e948e3fa52983fe8dee3cacd990a72989a39f539973f07fca0f2065dc7a0577dR15 
 This might not be the ideal solution, we can follow up with changes to this behavior.